AUDIT_CATALOG.md

```diff
@@ -3672,6 +4417,34 @@ gcloud eventarc triggers create my-trigger \
 - `google.cloud.oslogin.v1beta.OsLoginService.ContinueSession`
 - `google.cloud.oslogin.v1beta.OsLoginService.StartSession`
 
+### undefined
```

Comment:
I just noticed this display name for privateca.googleapis.com wasn't added. I have created a CL (cl/368061222) to fix this. The display name will be added the next time when Catalog Builder Runs (Apr 15th)